### ISSUE TITLE
getting blizzard UI blocked error

### ISSUE THREAD

**Issue by Brandonp1990** [tullamods/Dominos#713]
**When reporting an issue, please try to reproduce the issue with just Dominos and Dominos_Config running.** Dominos reuses a good chunk of the stock UI, and other popular addons, like Move Anything, like to take control of those parts, too.

**Addon Version**
Dominos version: 10/0/14-retail

**World of Warcraft Version**
 Retail, 

Consistently getting the blizzard error message that the addon has been blocked by blizzard UI and I should choose to ignore it or disable the addon.

**Comment by nick-telsan** [comment]
Getting the same error. Seemed to happen after dismounting from dragon riding, but haven't been able to pin point where it occurred.

```
3x [ADDON_ACTION_BLOCKED] AddOn 'Dominos' tried to call the protected function 'DominosActionButton3:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:412: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:1203: in function <FrameXML/ActionButton.lua:1200>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:266: in function <FrameXML/SecureHandlers.lua:263>
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:322: in function <FrameXML/SecureHandlers.lua:306>
[string "=(tail call)"]: ?
```

**Comment by Aideenwolf** [comment]
Sorry to bring a "dead" thread/bug report. I'm running Dominos 10.1.0, BE 4.5.10

So i noticed something similar to this but it's doing the same for BrokerEverything - Since the error is pointing at Dominos i'm putting it here too.

I've noticed that I get this 100% of the time when I swap a talent in a dungeon/raid then try to hit a mob. The "Action Button" changes based on whatever button I hit.
Either way - reload fixes the issue.


1x [ADDON_ACTION_BLOCKED] AddOn 'Broker_Everything' tried to call the protected function 'DominosActionButton4:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:335: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:395: in function <FrameXML/ActionButton.lua:356>
[string "=[C]"]: in function `Update'
[string "@FrameXML/ActionButton.lua"]:347: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:1189: in function <FrameXML/ActionButton.lua:1186>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:266: in function <FrameXML/SecureHandlers.lua:263>
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:322: in function <FrameXML/SecureHandlers.lua:306>
[string "=(tail call)"]: ?

Locals:
_ = Frame {
 RegisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 0 = <userdata>
 UnregisterEvent = <function> defined @!BugGrabber/BugGrabber.lua:486
 SetScript = <function> defined @!BugGrabber/BugGrabber.lua:486
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
 ADDON_ACTION_BLOCKED = <function> defined @!BugGrabber/BugGrabber.lua:552
 ADDON_ACTION_FORBIDDEN = <function> defined @!BugGrabber/BugGrabber.lua:552
 PLAYER_LOGIN = <function> defined @!BugGrabber/BugGrabber.lua:546
 LUA_WARNING = <function> defined @!BugGrabber/BugGrabber.lua:561
 ADDON_LOADED = <function> defined @!BugGrabber/BugGrabber.lua:506
}

**Comment by Tuller** [comment]
Duplicate of #701
